### DISCUSSION TITLE
XCH Balance are all gone after installed v1.3.3

### DISCUSSION THREAD

**Discussion by Ratchanida** [Chia-Network/chia-blockchain#11114] [Support]
I updated to v. 1.3.3.  The wallets are synced but all my XCH balance which I have about 10 XCH are all gone as well as my unclaimed reward on my solo pool is gone. Please help. Thanks

**Comment by 7by24** [comment]
you may need to track the transactions of your address first via https://www.chiaexplorer.com/blockchain/search

**Reply by Ratchanida** [reply]
Thanks.  Instead, I deleted the old db folder \.chia\mainnet\wallet\db and let it start syncing from 0.  I got all my xch back including unclaimed reward.
